coalib/bearlib/languages/definitions/BlitzMax.py

```diff
@@ -0,0 +1,6 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class BlitzMax:
```

Comment:
How different from  BlitzBasic. Maybe pygments delegation can be used as part of heuristics for this.